notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb

```diff
@@ -1040,7 +999,7 @@
       "source": [
         "### Train a BigQuery ML model with Explainability\n",
         "\n",
-        "Next, you train the same BigQuery ML model, but this time you enable Vertex AI Explainability on the model predictions by adding the option:\n",
+        "Next, you train the same BigQuery ML model, this time you enable Vertex AI Explainability on the model predictions by adding the option:\n",
```

Comment:
- change: "Vertex AI Explainability"
- to: "Vertex Explainable AI" (this is the product name)